contents/page-sections/hero-headers/with-content-on-left/index.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  Box,
+  Button,
+  Center,
+  Container,
+  Heading,
+  Text,
+  VStack,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+
+const HeroWithContentOnLeft: FC = () => {
+  return (
+    <Container
+      bgImage="url(https://images.unsplash.com/photo-1519389950473-47ba0277781c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80)"
+      bgSize="cover"
+      bgPosition="center"
+      minH="fit-content"
+      h="full"
+    >
+      <Box position="absolute" inset={0} zIndex="0" bg="blackAlpha.700" />
+      <VStack as={Center} px="xl" h="full" zIndex="1" py="xl" gap="lg">
```

Comment:
Buttons have been crushed.

<img width="1436" alt="image" src="https://github.com/user-attachments/assets/12cb3e87-daf0-4bdf-89bc-8b3fda6c188d">

Reply:
The button does not seem to be crushed on my end. Do you know the cause?
![image](https://github.com/user-attachments/assets/3918b23e-dff2-4152-b9b3-09cdba973169)